pokemon-persistence/pom.xml

```diff
@@ -13,6 +13,11 @@
     <name>Pokemon persistance</name>
 
     <dependencies>
+        <dependency>
+            <groupId>cz.muni.fi.pa165</groupId>
+            <artifactId>pokemon-api</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
use ${parent.project.groupId} and ${parent.project.version} respectively for dependencies on our other modules.